### ISSUE TITLE
[Enhancement]: Show Abriged Icon in Shelf

### ISSUE THREAD

**Issue by mutschler** [advplyr/audiobookshelf#2833]
### Describe the feature/enhancement

Like for Explicit Books: The icon in shown in the List views. However the Abridged Icon is only shown on the Book Detail view.
Would be great have the Icon in the list view as well so it's easier to spot if there's an Abridged and normal version of the book.

**Comment by advplyr** [comment]
I'm not sure how to show NOT abridged in the UI

**Comment by nichwall** [comment]
I think the "not abridged" is for the filter, like how there is "Has ebook" and "Has no ebook", but there is currently only an "Abridged" filter in the web app. This is separate to adding the "Abridged" indicator under to book next to the "Explicit" E icon.

**Comment by advplyr** [comment]
Yeah for the filter I was looking at where that would be shown in the UI. I guess it would have to be a submenu like ebooks but with only 2 options.

**Comment by Demian98** [comment]
@advplyr in my opinion, the abridged information is quite important for the listener. It would be great if it is not just display as a small icon. It should also be in the details list:
![grafik](https://github.com/advplyr/audiobookshelf/assets/77210091/825ea532-5f25-41e8-ad01-6bab6300d738)

What do you think about this?

**Comment by advplyr** [comment]
I don't like putting it there because I don't think boolean fits in that detailed list view.
What would it look like?
Abridged: Yes/No?

I believe Audible or one of the book providers puts (Abridged) in the title of the book. You could also do it that way.
